.github/workflows/release.yml.bk

```diff
@@ -0,0 +1,24 @@
+name: Release
+
+# Controls when the action will run.
+on:
+  push:
+    # Pattern matched against refs/tags
+    tags:
+    - '*'           # Push events to every tag
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: Deployment to WP SVN (tag only)
+    steps:
+    - name: Checkout from GIT
+      uses: actions/checkout@v2
+
+    - name: Run svn commit script
+      run: sh bin/commit-svn
```

Comment:
`bin/commit-svn` requires some arguments: `$TRAVIS_TAG --stable`. Is that doable here?